src/app/shell/personal-cabinet/provider/create-workshop/create-about-form/create-about-form.component.ts

```diff
@@ -17,8 +17,7 @@ import { MUST_CONTAIN_LETTERS } from 'shared/constants/regex-constants';
 @Component({
   selector: 'app-create-about-form',
   templateUrl: './create-about-form.component.html',
-  styleUrls: ['./create-about-form.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush
```

Comment:
It is better to leave ChangeDetectionStrategy.OnPush, just force change detector in those places where it is needed